### DISCUSSION TITLE
Lesson 11: IPFS Install Issues

### DISCUSSION THREAD

**Discussion by brownandrew1390** [smartcontractkit/full-blockchain-solidity-course-py#503] [Q&A] [answered]
Having some issues trying to install IPFS on my machine.  I was able to follow all the steps with no errors until I got to step 7, where I try to load my Powershell profile that I had created:

`& $profile`   

Here is the error that I got:

```
& : File C:\Users\Andrew\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1 cannot be loaded 
because running scripts is disabled on this system. For more information, see about_Execution_Policies    
at https:/go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:3
+ & $profile
+   ~~~~~~~~
    + CategoryInfo          : SecurityError: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess
```
I'm not too familiar with Powershell, I mostly use Bash.  Let me know if I'm missing something here!

**Comment by cromewar** [comment, marked as answer]
Hello @brownandrew1390 as a security measure windows blocks script execution by default so to allow them you should:

1. Open powershell as admin.
2. paste this code: `set-executionpolicy remotesigned`
3. hit enter.
4. close all the terminals you have opened. 
5. Open a new one and try again where you left.

**Reply by brownandrew1390** [reply]
Excellent. Again, @cromewar, thank you, you are the man!

**Reply by cromewar** [reply]
I'm glad it helped, feel free to make any other questions
